doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -3,161 +3,177 @@
+  ``util/setchplenv.bash`` instead of
+  ``util/quickstart/setchplenv.bash``.  This will set up your
+  environment to use Chapel in the preferred configuration.  Note that
+  building this configuration involves compiling third-party packages,
+  which will increase the overall build time.
 
-   Note that the environment settings from ``util/setchplenv.bash`` will not persist beyond this terminal session.
-   You can choose to source ``setchplenv.bash`` whenever you want to use relevant commands.
-   If you want these environment settings to persist for future terminal sessions,
-   copy the commands from the Recommended Settings in :ref:`chplenv.rst <readme-chplenv.recommended_settings>` into your ``~/.bashrc`` file.
-   You can also store Chapel configuration settings in a :ref:`chplconfig <readme-chplenv.chplconfig>` file instead of your ``~/.bashrc``.
+  .. code-block:: bash
 
-   See :ref:`chplenv.rst <readme-chplenv>` for a complete description of
-   Chapel's configuration variables, what they mean, and how they
-   can be set.
+     # Set environment variables to preferred configuration
+     source util/setchplenv.bash
+
+     # re-build Chapel
+     make
+
+     # compile a sample program
+     chpl examples/hello3-datapar.chpl
+
+     # run the sample program
+     ./hello3-datapar
+
+  If you run into any portability issues, please see
+  :ref:`readme-bugs`.
+
+.. _using-chapel-in-another-shell:
+
+Using Chapel in a Different Shell or Terminal
+---------------------------------------------
+
+Note that in both the Quickstart and preferred modes above, any
+environment settings made by ``setchplenv.bash`` will not persist
+beyond your current shell / terminal session.  To use Chapel from a
+different shell or terminal, you will need to either re-``source`` the
+``setchplenv.bash`` script that you used when building Chapel or set
+up your environment to support additional shells automatically.  See
+:ref:`using-chapel-in-other-sessions` for details.
 
 
 Using Chapel in Multi-Locale Mode
 ---------------------------------
 
-All of the instructions above describe how to run Chapel programs
-in a single-locale (shared-memory) mode. To run using multiple
-locales (distributed memory), please refer to
-:ref:`multilocale.rst <readme-multilocale>`.
+All of the instructions above describe how to run Chapel programs in a
+single-locale (shared-memory) mode. To run using multiple locales
+(multiple compute nodes with distributed memory), please refer to
+:ref:`readme-multilocale`.
+
 
-Performance
------------
+Notes on Performance
+--------------------
 
-If you plan to do performance studies of Chapel programs, be sure to use the
-full-featured version from using-a-more-full-featured-chapel_ above and see
-https://chapel-lang.org/performance.html for performance tips.
+If you plan to do performance studies of Chapel programs, be sure to
+use the full-featured version above and see
+https://chapel-lang.org/perf-tips.html for valuable tips.
```

Comment:
Based on how `--fast` can be overlooked, we could consider mentioning it here, and referring to perf-tips for further notes (which may not be much today, but still)

Reply:
OK, so maybe:

> If you plan to do performance studies of Chapel programs, be sure to
> use the full-featured version above, to compile with `--fast` once
> your program is correct, and to see
> https://chapel-lang.org/perf-tips.html for other valuable tips.

Reply:
On second thought, I dropped `valuable` now that `--fast` is here (the clearly most valuable).